brigade.js

```diff
@@ -181,6 +181,10 @@ function publish(e, p) {
 
   goPublish.env.AZURE_STORAGE_CONNECTION_STRING = p.secrets.azureStorageConnectionString;
   goPublish.tasks.push(
+    "apk update && apk add py-pip && \
```

Comment:
if we just need the azure cli for publish, then I recommend using 

https://github.com/carolynvs/az-cli

Though for some of our lightweight docker images I think it's installed by default.

Reply:
I'd be down to switch up to using that cli.  It's not necessarily updated to the latest az cli codez routinely, right?  But perhaps for our purposes, it doesn't matter.

Alternatively, we can add `az` to the `go-dind` image used here.  Actually, if this `go-dind` image garners favor, we should probably move it to this `deislabs` org and potentially remove `kubectl`.  Then, we could have a porter CI-centric Dockerfile here based on `go-dind` but bringing in `kubectl`, `az` and any other dependencies that might come later on?  What do you think?

Reply:
The az-cli repo I linked is a go implementation of just a few select commands from the full az cli which is written in python. It is a bit smaller and easier to install, so it's friendlier to get into a docker image is all. But because it's not actually using the az cli, it's not quite the same concern when it comes to using the latest code. More like, do you trust the code that I wrote and tested really.

I have been using it for a few open source projects, like service catalog, minibroker, etc to handle deployments to azure storage.

Reply:
Good call!  Updated to use this `az`.  Saves so much time and just what we need here.  Thanks!

Reply:
Sorry it was borked the first time you tried it. Thanks for showing me how to regex. 😀